ext_tables.sql

```diff
@@ -81,3 +81,18 @@ CREATE TABLE tx_examples_haiku (
 
 	PRIMARY KEY (uid)
 );
+
+# Table structure for table 'tx_examples_log'
+#
+# The KEY on request_id is optional
+#
+CREATE TABLE tx_examples_log (
```

Comment:
Shouldn't there be a configuration to use this table?

Reply:
You are right, I added the configurations of example logs.